### PR TITLE
Removed ITMGeolocationFragment by using lifecycle observation instead.

### PR DESCRIPTION
**Pull request by toddsouthenbentley**

### CODE REVIEW

File: mobile-sdk/src/main/java/com/github/itwin/mobilesdk/ITMGeolocationManager.kt [hunk 1]
```diff
@@ -35,11 +43,11 @@ import kotlin.concurrent.schedule
 /**
  * Class for the native-side implementation of a `navigator.geolocation` polyfill.
  *
- * __Note:__ [ITMGeolocationManager] must be bound to an [ITMGeolocationFragment]-based fragment for showing permission
- * and service dialogs and handling responses. This happens automatically in the [ITMGeolocationFragment] constructor.
+ * __Note:__ [ITMGeolocationManager] must be bound to a [ActivityResultCaller] for showing permission
```
Comment (tcobbs-bentley) [inline mobile-sdk/src/main/java/com/github/itwin/mobilesdk/ITMGeolocationManager.kt:46]: "a" should be "an" (before [ActivityResultCaller]).

File: mobile-sdk/src/main/java/com/github/itwin/mobilesdk/ITMGeolocationManager.kt [hunk 2]
```diff
@@ -109,13 +117,35 @@ class ITMGeolocationManager(private val appContext: Context, private val webView
         return GeolocationPosition(coordinates)
     }
 
+    var webView: WebView? = null
```
Comment (tcobbs-bentley) [inline mobile-sdk/src/main/java/com/github/itwin/mobilesdk/ITMGeolocationManager.kt:120]: Needs to be documented.

**Comment by tcobbs-bentley** [comment]
I finished my first pass with some comments.

File: mobile-sdk/src/main/java/com/github/itwin/mobilesdk/ITMGeolocationManager.kt (hunk 2 above)
Comment (toddsouthenbentley) [reply mobile-sdk/src/main/java/com/github/itwin/mobilesdk/ITMGeolocationManager.kt:122]: Done

File: mobile-sdk/src/main/java/com/github/itwin/mobilesdk/ITMGeolocationManager.kt (hunk 1 above)
Comment (toddsouthenbentley) [reply mobile-sdk/src/main/java/com/github/itwin/mobilesdk/ITMGeolocationManager.kt]: Done

**Review by tcobbs-bentley — APPROVED** [review]

**Pull request merged**